appengine/hello_world/app.yaml

```diff
@@ -16,4 +16,16 @@
 runtime: ruby
 env: flex
 entrypoint: bundle exec ruby app.rb
+
```

Comment:
Did you mean to put in within the region tag? If not can you move out of the resource and manual scaling configuration out of the region tags for simplicity.

Reply:
Yes, I want this to be part of the config that is pulled into the quickstart docs.

Reply:
FWIW, documentation should provide the description below and not be in the app.yaml. In case it needs to be changed it can be updated directly in the docs instead of the app.yaml.

Reply:
Actually, I think it makes more sense to have it embedded within the code than within the docs. The code and the side affect of running it could be the same regardless of if your discovery point is the docs. We don't want to surprise users with unexpected bills regardless of how they come across the sample.

Reply:
Understood, thanks for sharing your perspective.